composer.json

```diff
@@ -50,7 +50,8 @@
         "mockery/mockery": "^1.6",
         "phpunit/phpunit": "^9.6",
         "san/san-session-toolbar": "*",
-        "bamarni/composer-bin-plugin": "^1.8"
+        "bamarni/composer-bin-plugin": "^1.8",
+        "rector/rector": "^1.0"
```

Comment:
Should we keep this out of the main dependency tree for now?

Reply:
Yeah i was going to remove that but missed it thanks.